src/FlipCounter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      x: 0
+  
+    };
+  }
+  add = (num) => {
+    this.setState({ x: parseInt(this.state.x) + 1 });
+    console.log("add activated");
+    console.log(this.state.x);
+  };
+ 
+ 
+ 
+  myChangeHandler = (event) => {
```

Comment:
It is common in React world to name event handlers as `handle + enentName`. e.g. `onSubmit` event handler it is common to name `handleSubmit`